jetbrains/src/main/kotlin/com/sourcegraph/cody/config/CodySettingsFileChangeListener.kt

```diff
@@ -20,12 +20,10 @@ class CodySettingsFileChangeListener(private val project: Project) : FileDocumen
         LocalFileSystem.getInstance()
             .refreshAndFindFileByNioFile(ConfigUtil.getSettingsFile(project))
     if (currentFile == configFile) {
-      // TODO: it seams that some of the settings changes (like enabling/disabling
-      // autocomplete)
-      // requires agent restart to take effect.
+      // TODO: it seams that some of the settings changes (like enabling/disabling autocomplete)
+      //       requires agent restart to take effect.
```

Comment:
```suggestion
      // TODO: it seems that some of the settings changes (like enabling/disabling autocomplete)
      //       require agent restart to take effect.
```

I know you're just moving this; grammar fix suggested inline.

This is a weak TODO because it misses the two most important points of a to-do:
1. What to do.
2. When to do it (usually, what's blocking doing it now.)

If we can improve this by adding those, that would be nice.

Reply:
good point 

The comment updated. An issue opened.

https://linear.app/sourcegraph/issue/CODY-5682/handle-all-agent-configuration-changes-without-a-restart